mantidproject/mslice
# Colorbar range and scale type editable

The colorbar scale minimum and maximum values can now be changed via the plot options dialog's corresponding text boxes.

Additionally, the colorbar scale can also be changed to logarithmic by ticking a new "logarithmic" checkbox in the plot options dialog.

**To test:**

Ensure that the minimum and maximum scale can be adjusted, as well as the colorbar and graph change to logarithmic when selected.

Fixes #80
